types/ui-grid/index.d.ts

```diff
@@ -3799,7 +3799,7 @@ declare namespace uiGrid {
          * If suppressRemoveSort is also set, the unsorted state will be skipped even if it is listed here. Each direction may
          * not appear in the list more than once (e.g. [ASC, DESC, DESC] is not allowed), and the list may not be empty.*
          */
-        sortDirectionCycle?: Array<IUiGridConstants>;
+        sortDirectionCycle?: Array<null | string>;
```

Comment:
I'm curious why this was merged as-is.  Are the proposed changes really that difficult to make?